views/stanford/about_get_involved.pug

```diff
@@ -2,29 +2,126 @@ extends ../layout
+                              with the goal of creating the best music virtual assistant that can control their Spotify music through any device. 
+                              What they built went beyond Alexa or Google Home, allowing for actions like the ability to change playlists.
+                    li.glide__slide
+                      div.carousel-card(style={ background: 'url(' + Config.ASSET_CDN + '/images/multi_user_almond.jpg' + ') no-repeat center', 'background-size': 'cover' })
+                        a(href="https://www.youtube.com/watch?v=LFOB_AUmreE&feature=emb_title", target="_blank")
+                          div.carousel-card-gradient
+                            h3.
+                              Multi-user Almond
+                            p.carousel-card-subtitle.
+                              Sharing a smart speaker among multiple people in the household is useful but at the same time raises privacy and parental control issues. 
+                              This project showed that Almond can be used to identify users with their voice and provide fine-grained access control based on the detected identity. 
+                    li.glide__slide
+                      div.carousel-card(style={ background: 'url(' + Config.ASSET_CDN + '/images/mmi.jpg' + ') no-repeat center', 'background-size': 'cover' })
+                        a(href="https://www.youtube.com/watch?v=-8XIzZufA7M", target="_blank")
+                          div.carousel-card-gradient
+                            h3.
+                              Multi-modal Interface
+                            p.carousel-card-subtitle.
+                              Jackie Yang built a multi-modal interface that allows you to use Almond to make complex queries while interacting with someone over chat.
+                    li.glide__slide
+                      div.carousel-card(style={ background: 'url(' + Config.ASSET_CDN + '/images/sports_wikidata.jpg' + ') no-repeat center', 'background-size': 'cover' })
+                        a(href="https://www.youtube.com/watch?v=DJqLgOdEEcw&feature=youtu.be", target="_blank")
+                          div.carousel-card-gradient
+                            h3.
+                              Sports Wikidata
+                            p.carousel-card-subtitle.
+                              High school student Ryan Cheng connected Almond to an API with NFL data, allowing the assistant to return information about team rankings, scores, and more.
 
-              p.text-center
-               a(href='https://forms.gle/Q1m5uZDF8ZyRbdSw7',rel='noopener').btn.btn-primary= _("Sign up to be part of our community")
+        div.container
+          div.row#jumbotron-prompt
+            div.col-xs-12.col-sm-8.col-sm-offset-2.col-md-6.col-md-offset-3
+              h4
+                strong
+                  a(href="https://almond.stanford.edu/doc/getting-started.md")=_("Getting Started »")
```

Comment:
Is the chiron / quotation mark intended?

Reply:
Yes. Further emphasis on link